src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -120,6 +126,19 @@
         </div>
     {% endif %}
 
+    {% if collector.firewall|default([]) is not empty %}
+        <h2>Security Firewall</h2>
+
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.firewall.name|default('unknown') }}</span>
+                <span class="label">Firewall Name</span>
```

Comment:
It would be nice to have booleans (`security`, `anonymous`, `sateless`?) and maybe the context as metrics.

Not sure if the `request_matcher` is that useful in the parameters bag but maybe the listeners would?